src/libs/checkForUpdates.ts

```diff
@@ -0,0 +1,19 @@
+const UPDATE_INTERVAL = 1000 * 60 * 60 * 8;
+
+type CheckForUpdatesParams = {
```

Comment:
```suggestion
type PlatformSpecificUpdater = {
```

We could use a more specific name for this

Reply:
Hi @arosiclair, earlier @fabioh8010 asked me to change from this which you suggested to what is now, can we make an agreement what name we want to use?

CC: @fabioh8010

Reply:
Generally speaking our style guidance is to use meaningful names for variables and functions (eg: no `onClick()` or `onSubmit()` it should be `doSomethingImportant()`). We should aim to do the same thing with type names and `PlatformSpecificUpdater` is a lot more descriptive than `CheckForUpdatesParams`.

Reply:
@arosiclair fixed